Document/0x06c-Reverse-Engineering-and-Tampering.md

```diff
@@ -316,4 +316,93 @@ Cycript tricks:
 
 http://iphonedevwiki.net/index.php/Cycript_Tricks
 
-<!-- TODO [Develop section on Frida] -->
+#### Frida
+
+It’s Greasemonkey for native apps, or, put in more technical terms, it’s a dynamic code instrumentation toolkit. It lets you inject snippets of JavaScript or your own library into native apps on Windows, macOS, Linux, iOS, Android, and QNX. Frida also provides you with some simple tools built on top of the Frida API. These can be used as-is, tweaked to your needs, or serve as examples of how to use the API.
```

Comment:
This is copy paste from the Frida website https://www.frida.re/docs/home/. 

Please put a link for the reference and please rephrase it with your own words.